src/python/pants/option/global_options.py

```diff
@@ -168,7 +168,25 @@ def register_bootstrap_options(cls, register):
             "--level",
             type=LogLevel,
             default=LogLevel.INFO,
-            help="Set the logging level.",
+            help="Set the logging level. The logging levels are one of: "
+            '"error", "warn", "info", "debug", "trace".',
+        )
+        register(
+            "--show-log-target",
+            type=bool,
+            default=False,
+            advanced=True,
+            help="Display the target where a log message originates in that log message's output.",
```

Comment:
Maybe add: "This can be helpful when paired with the option `--log-levels-by-target`"